modules/ossm-about-concepts-resources.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+// about/ossm-about-concepts-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-concepts-resources_{context}"]
+= {SMProductName} resources
+
+//Should each subject/topic be its own module in the about-concepts-assembly? Kiali module, traces modules, metrics module, Argo roll outs module, cert-manager module?
+//very rough draft, copy/pasted from upstream to create the PR to have a visual of what just upstream looks like in prod docs.
+//need Kiali, tracing, metrics, cert-manager, and Argo rollouts
+
+{SMProductName} Operator manages the lifecycle of your Istio control planes. Instead of creating a new configuration schema, {SMProduct} Operator APIs are built around Istio's Helm chart APIs.
+
+//All installation and configuration options that are exposed by Istio's Helm charts are available through the {SMProduct} Operator Custom Resource Definition (CRD) `values` fields.
+
+[NOTE]
+====
+* Though {SMProductName} APIs are built around Istio's Helm chart APIs, Helm charts are not supported.
+* All installation and configuration options that are exposed by Istio's Helm charts are available through the {SMProductName} Custom Resource Definition (CRD) `values` fields.
+* CRDs are supported.
```

Comment:
What CRDs?

Reply:
Custom Resource Defintion

Reply:
What CRDs are supported? The Istio CRDs? I'm not sure specifically _what_ CRDs are supported here.

Reply:
@longmuir , are there other CRDs that are supported, or just OpenShift Service Mesh CRDs?

Reply:
I'm not sure what the last bullet here "*CRDs are supported" refers to...it looks as if it's missing something before it. There are a lot of CRDs we support under "OpenShift Service Mesh CRDs"... the sail operator ones (Istio, IstioRevision), Istio's CRDs (VirtualService, etc),.. even Kiali's CRDs (Kiali, OSSMConsole). 

It's on us to call out what we don't support. Probably the only one I can think of that we don't support directly is the upstream `IstioOperator` resource, as we have the `Istio` resource (though I believe this is captured).

Reply:
I can take that last bullet point out.